src/Compilers/CSharp/Portable/Syntax/SyntaxNodeRemover.cs

```diff
@@ -260,6 +288,19 @@ public override SeparatedSyntaxList<TNode> VisitList<TNode>(SeparatedSyntaxList<
                 return list;
             }
 
+            private bool ContainsEndOfLine(SyntaxTriviaList triviaList)
```

Comment:
:bulb: Several other kinds of trivia have a newline built-in. StyleCop Analyzers uses a [`HasBuiltinEndLine`](https://github.com/DotNetAnalyzers/StyleCopAnalyzers/blob/6d80369e41d01fc31732ba1d4dbdf0884f7f4b3d/StyleCop.Analyzers/StyleCop.Analyzers/Helpers/TriviaHelper.cs#L388-L393) helper method for this function.